### ISSUE TITLE
[Spark-Debug] For killed job, the submission toolwindow shows `JsonSyntaxException`

### ISSUE THREAD

**Issue by wezhang** [Microsoft/azure-tools-for-java#546]
If a debug job is killed, the toolwindow would show:

```
Error : Failed to getting running log. Exception : com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 2 path $
```

**Comment by wezhang** [comment]
We can suspend the log printing when getting the job status is `KILLED`  as the following snapshot:

![image](https://cloud.githubusercontent.com/assets/7385667/26383788/711f31b2-4067-11e7-9740-03026931ee75.png)

**Comment by wezhang** [comment]
fixed by #837

**Comment by ruoxinCheng** [comment]
Verified it.
